core/src/main/java/io/questdb/cairo/mv/MatViewGraph.java

```diff
@@ -154,6 +154,27 @@ public void removeView(TableToken matViewToken) {
         }
     }
 
+    public void updateToken(TableToken updatedToken) {
+        final MatViewDefinition viewDefinition = definitionByTableDirName.get(updatedToken.getDirName());
+        if (viewDefinition != null) {
+            viewDefinition.updateToken(updatedToken);
+            MatViewDependencyList viewList = dependentViewsByTableName.get(viewDefinition.getBaseTableName());
```

Comment:
base table name is case-sensitive here, might break

Reply:
```
dependentViewsByTableName = new ConcurrentHashMap<>(false);
```

`isCaseSensitive = false` constructor parameter means case insensitive map.